packages/core-mobile/app/new/common/consts/screenOptions.tsx

```diff
@@ -25,8 +25,7 @@ const commonNavigatorScreenOptions: StackNavigationOptions = {
 
 export const stackNavigatorScreenOptions: StackNavigationOptions = {
   ...commonNavigatorScreenOptions,
-  headerTransparent: true,
-  headerBackground: () => <BlurredBackgroundView />
+  headerTransparent: true
```

Comment:
I removed this as it looks like we are setting the blurred background view within useFadingHeaderNavigation() already